app/routes/goal_routes.py

```diff
@@ -1 +1,74 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, make_response, abort, request, Response
```

Comment:
`abort` and `Response` get imported but are never accessed in `goal_routes.py`. Take care to remove unused imports from a file.